source/write/bulk-write.txt

```diff
@@ -15,43 +15,50 @@ Bulk Write Operations
    :values: reference
 
 .. meta::
-   :keywords: insert, update, replace, code example
+   :keywords: insert, update, replace, code example, multiple changes
 
 Overview
 --------
 
-In this guide, you can learn how to perform multiple write operations
-in a single database call by using **bulk write operations**.
+In this guide, you can learn how to perform multiple write operations in
+a single database call by using **bulk write operations**.
 
-Consider a scenario in which you want to insert a document into a collection,
+Consider a scenario in which you want to insert a document,
 update multiple other documents, then delete a document. If you use
-individual methods, each operation requires its own database call. Instead,
-you can use a bulk operation to reduce the number of calls to the database.
+individual methods, each operation requires its own database call.
 
-Sample Data
-~~~~~~~~~~~
+By using a bulk write operation, you can perform multiple write operations in
+fewer database calls. You can perform bulk write operations at the following levels:
 
-The examples in this guide use the ``restaurants`` collection in the ``sample_restaurants``
-database from the :atlas:`Atlas sample datasets </sample-data>`. To access this collection
-from your PHP application, instantiate a ``MongoDB\Client`` that connects to an Atlas cluster
-and assign the following value to your ``$collection`` variable:
+- :ref:`Collection <php-coll-bulk-write>`: You can use the 
+  ``MongoDB\Collection::bulkWrite()`` method to perform bulk write
+  operations on a single collection. This method makes a database call
+  for each type of write operation. For example, the method can perform
+  multiple update operations in one call, but makes two separate calls to
+  the database for an insert operation and a replace operation.
```

Comment:
Today I learned that collection bulk write is a driver implementation only to batch individual commands. Confirmed by the spec.